.jenkins/pytorch/test.sh

```diff
@@ -278,6 +278,12 @@ test_inductor_benchmark() {
     --device cuda --inductor --amp $PARTITION_FLAGS  --output "$TEST_REPORTS_DIR"/inductor_training_$1.csv
   # shellcheck disable=SC2086
   python benchmarks/dynamo/check_csv.py -f "$TEST_REPORTS_DIR"/inductor_training_$1.csv
+  # Check training with symbolic shapes (not actually inductor)
+  # shellcheck disable=SC2086
+  python benchmarks/dynamo/$1.py --ci --training --accuracy --dynamic-shapes \
```

Comment:
I am curious how much time this adds. Since the `sm86` instances are more stable now, we could make a small subset of the benchmark tests to run for every PR, and include more tests for nightly, such as dynamo-eager, aot-eager, aot-eager-dynamic, etc.

Reply:
I can pull numbers for you once it works. I'm guessing it is comparable to the other two runs, so this represents a 50% increase in runtime. How can we make a call on this?

Reply:
Timing wise, on this PR the inductor test step took: 1h 7m 45s, 1h 26m 45s, 1h 12m 26s, 1h 0m 23s.

Comparison with https://hud.pytorch.org/pytorch/pytorch/commit/c6c2de586d7f6ecd6a3eb5139870824f33a1f916 the inductor test step takes 49m 19s, 57m 46s, 43m 53s, 46m 40s

So that's on order of +20min